SetupDataPkg/Docs/Profiles/Overview.md

```diff
@@ -99,3 +99,48 @@ knobs that can be iterated through and applied to the `CacheValueAddress` of eac
 Alternatively, if the generic profile is chosen, ActiveProfileIndexSelectorLib will return MAX_UINT32 to indicdate the
 gProfileData structure is not used for this boot and instead only the defaults in gKnobData (and possibly any
 overrides found in variable storage) will be used.
+
+### Platform Usage Recommendations
+
+#### Generate Profile CSV Files
+
+There are 2 ways to generate the CSV files:
+
+- Through ConfigEditor UI tool:
+
+  - Load the XML configuration file
+  - Make any changes
+  - Save the CSV file by selecting the "Save Config Changes to Change File" button. Show below:
+
+    ![Profile Through ConfigEditor UI tool](Images/profile_ui.png)
+
+- Through the command line:
+
+  - Run the following command:
+
+    ```bash
+    GenNCCfgData  GENCSV  XmlFile[;BinFile]   CsvOutFile
+    ```

```

Comment:
I think the command is simple enough, but you might also want to consider adding an actual example while you're here.

Reply:
Thanks, updated.